src/assets/css/style.css

```
@@ -0,0 +1,163 @@
+  --border-width-1_5: 1.5px;
+  --border-width-2_5: 2.5px;
+
+  /* ---- Transition durations ---- */
+  --transition-duration-hover: 200ms;
+
+  /* ---- Max width ---- */
+  --max-width-8xl: 90rem;
+
+  /* ---- Font families ---- */
+  --font-sans:
+    -apple-system, ui-sans-serif, system-ui, BlinkMacSystemFont, "Segoe UI",
```

Stylelint (17.11.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)